ios/engine/KMEI/KeymanEngine/INSTRUCTIONS

```diff
@@ -7,8 +7,7 @@ Creating the Library
 3) select the "Keyman-lib" target, and build for device
```

Comment:
- [ ] Either change this to be up-to-date with the framework or include the new details within the general readme.md.